lecture04/Stream/src/EnhancedForSample.java

```diff
@@ -0,0 +1,22 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class EnhancedForSample {
+
+    public static void main(String[] args) {
+
+        List<String> names = List.of("yamada", "tanaka", "yasuda", "suzuki", "iida");
+        List<String> result = new ArrayList<>();
+
+        System.out.println("[拡張for文]名前にaが含まれるかを判定します。");
+        int nameIndex = 0;
+        for (String name : names) {
+            System.out.print(names.get(nameIndex++) + "：");
+            if (name.contains("a")) {
+                System.out.println(result.add(name));
```

Comment:
Listのaddメソッドでtrue/falseを判定しているのはなぜでしょうか？
特に意図がないのであればtrueを出力するで良いと思いました。
```suggestion
                System.out.println(true);
```

Reply:
特に意図はない ＋ true にした方が可読性があがるため、対応しました！
[752bbc6](https://github.com/aoha968/Java2211/pull/4/commits/752bbc681f172c850e9c626902006c6bbb0045e8)